zio/zngio/index.go

```diff
@@ -51,12 +61,14 @@ func (ti *TimeIndex) NewReader(f *os.File, zctx *resolver.Context, span nano.Spa
 type indexReader struct {
 	Reader
 	io.Closer
-	start   nano.Ts
-	end     nano.Ts
-	parent  *TimeIndex
-	marks   []mark
-	lastSOS int64
-	lastTs  nano.Ts
+	start         nano.Ts
+	end           nano.Ts
+	parent        *TimeIndex
+	order         Ordering
+	marks         []mark
+	lastSOS       int64
+	lastTs        nano.Ts
```

Comment:
nit: it might be more descriptive to call this `lastSoSTs` instead of `lastTs`

Reply:
But that's not what this is -- it is the timestamp of the last read record of any sort.

Reply:
oh nvm